rundeckapp/src/test/groovy/rundeck/services/ScheduledExecutionServiceSpec.groovy

```diff
@@ -3036,6 +3046,52 @@ class ScheduledExecutionServiceSpec extends Specification {
         false           | false            | true        | false
     }
 
+    @Unroll
+    def "nextExecutionTime project scheduled/executions enable/disable"() {
+        given:
+        setupDoValidate(true)
+        service.quartzScheduler = Mock(Scheduler)
+        service.quartzScheduler.getTrigger(_) >> Mock(Trigger){
+            getNextFireTime() >> new Date()
+        }
+
+        def projectMock = Mock(IRundeckProject) {
+            getProjectProperties() >> ['project.disable.schedule':projectScheduledEnabled,
+                                       'project.disable.executions': projectExecutionsEnabled]
+        }
+
+        service.frameworkService = Mock(FrameworkService) {
+            getFrameworkProject(_) >> projectMock
+        }
+
+        def job = new ScheduledExecution(
+                createJobParams(
+                        scheduled: true,
+                        scheduleEnabled: true,
+                        executionEnabled: true,
+                        userRoleList: 'a,b'
+                )
+        ).save()
+
+        when:
+        def result = service.nextExecutionTime(job)
+
+        then:
+        if(expectScheduled){
+            assertNotNull(result)
+        }else{
+            assertNull(result)
+        }
+
+
+        where:
+        projectScheduledEnabled   | projectExecutionsEnabled    | expectScheduled
```

Comment:
the variables seem to be named incorrectly, `projectScheduledDisabled` and `projectExecutionsDisabled` makes more sense